src/Config.php

```diff
@@ -1,51 +1,52 @@
 <?php
+
 namespace Pantheon\Terminus;
 
 use Dotenv\Dotenv;
 use Symfony\Component\Yaml\Yaml;
 
 class Config
 {
-  /**
-   * @var string[]
-   */
+    /**
```

Comment:
PSR-2 is being adopted in the new, 1.0 files and is linting the new files for this standard.